### ISSUE TITLE
Yuzu fails to configure due to Boost

### ISSUE THREAD

**Issue by Shoegzer** [yuzu-emu/yuzu#4293]
Yuzu commit a0ee597b19482ce4bfc6231ad54666e2a7f894b5 is failing to configure with cmake due to an issue with Boost.  The following is returned from console after a cmake is performed:

```
CMake Error at /usr/lib/x86_64-linux-gnu/cmake/Boost-1.71.0/BoostConfig.cmake:117 (find_package):
  Could not find a package configuration file provided by "boost_context"
  (requested version 1.71.0) with any of the following names:

    boost_contextConfig.cmake
    boost_context-config.cmake

  Add the installation prefix of "boost_context" to CMAKE_PREFIX_PATH or set
  "boost_context_DIR" to a directory containing one of the above files.  If
  "boost_context" provides a separate development package or SDK, be sure it
  has been installed.
Call Stack (most recent call first):
  /usr/lib/x86_64-linux-gnu/cmake/Boost-1.71.0/BoostConfig.cmake:182 (boost_find_component)
  src/common/CMakeLists.txt:192 (find_package)

```
It's possible this is due to a misconfiguration on the test system, which was recently upgraded from Linux Mint 19.3, but need to know if this is a known issue given similar linux compilation issues that have surfaced over the past week.

Using Linux Mint 20, kernel 5.4 with gcc 9.3.0.  

[CMakeOutput.log](https://github.com/yuzu-emu/yuzu/files/4906297/CMakeOutput.log)
[CMakeError.log](https://github.com/yuzu-emu/yuzu/files/4906298/CMakeError.log)

**Comment by lat9nq** [comment]
I just built yuzu-emu/yuzu with Linux Mint 20, same kernel and compiler version. Make sure you have updated packages, specifically `libboost-context1.71-dev`, as removing that package gives me the same error as you posted.

**Comment by Shoegzer** [comment]
Hey thanks @lat9nq.  Actually I was missing that package for some strange reason, I'm not entirely sure why though.  This was an upgrade from 19.3 and it must have been installed on the older version because I had been compiling yuzu before, so I suppose it was removed.  However, I made note of the dependencies for yuzu from before, and libboost-context1.71-dev wasn't one of them.  No matter, the issue is resolved thanks to your help.  Closing.
